utilities/merge_operators/cassandra/merge_operator.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * A MergeOperator for rocksdb that implements Cassandra row value merge.
+ * @author Chen Shen (chenshen@fb.com)
+ * Copyright 2017 Facebook
+ */
+
+#pragma once
+#include "rocksdb/merge_operator.h"
+#include "rocksdb/slice.h"
+
+namespace rocksdb {
+namespace cassandra {
+
+class CassandraValueMergeOperator : public MergeOperator {
+public:
+  static std::shared_ptr<MergeOperator> CreateSharedInstance();
+
+  virtual bool FullMerge(const Slice& key,
```

Comment:
`FullMerge` is deprecated. Would you be interested in converting to `FullMergeV2`?